library/std/src/sys/unix/process/process_common.rs

```diff
@@ -87,6 +87,7 @@ pub struct Command {
 }
 
 // Create a new type for argv, so that we can make it `Send` and `Sync`
+#[derive(Debug)]
 struct Argv(Vec<*const c_char>);
```

Comment:
This will show the raw pointer values, which aren't very useful.

Reply:
Removed thanks, wasn't sure how useful it was but thought it was good to include at first in case of very low level debugging.